src/solvers/dgsem_structured/dg_2d.jl

```diff
@@ -310,7 +310,10 @@ end
             u_ll = get_node_vars(u, equations, dg, i - 1, j, element)
             u_rr = get_node_vars(u, equations, dg, i, j, element)
 
-            for m in 1:nnodes(dg)
+            # Compute subcell volume operator for the finite volume flux,
+            # see equation (14) from the paper cited above.
+            # The `weights` correspond to the matrix M^{-1} and `derivative_matrix` is Δ.
+            for m in eachnode(dg)
```

Comment:
I'm actually a bit confused by this explanation. Doesn't the computation from (14) happens later? Maybe in `calc_volume_flux!`. I don't see, how `M^{-1}` corresponds to the weights and don't think the triangle is a matrix itself, rather triangle f^FV is a matrix.
In general, isn't the formula B.53 from the paper the perfect explanation for this computation?

<img width="835" height="212" alt="Image" src="https://github.com/user-attachments/assets/d19c87f6-16c0-436d-ba3b-a015eb111561" />

Reply:
True, this makes no sense since the mass matrix is inverted in (14). I will adapt the comment :+1: